testcases/kernel/controllers/memcg/stress/memcg_process_stress.c

```diff
@@ -92,13 +92,16 @@ int main(int argc, char *argv[])
 		interval = 1;
 
 	/* TODO (garrcoop): add error handling. */
+	/* TODO(mw): consider DRYing it */
 	memset(&sigint_action, 0, sizeof(sigint_action));
 	sigint_action.sa_handler = &sigint_handler;
-	sigaction(SIGINT, &sigint_action, NULL);
+	if (sigaction(SIGINT, &sigint_action, NULL) != 0)
+		errx(1, "sigint_action");
```

Comment:
This should be err, not errx (the former prints out the error; the latter does not).